pkg/network/ebpf/c/http.h

```diff
@@ -141,30 +141,32 @@ static __always_inline int http_begin_response(http_transaction_t *http, const c
 }
 
 static __always_inline void http_parse_data(char *p, http_packet_t *packet_type, http_method_t *method) {
-    if ((p[0] == 'H') && (p[1] == 'T') && (p[2] == 'T') && (p[3] == 'P')) {
+#define __builtin_const_strcmp(buf, constbuf) __builtin_memcmp(buf, constbuf, __builtin_strlen(constbuf))
```

Comment:
Have you examined the instructions to see if the compiler is smart enough to replace `__builtin_strlen` with a constant value?

Reply:
is the tests, `__builtin_strlen()` vs `constant value` generate the same numbers of instructions